### DISCUSSION TITLE
How to remap keys into the 'space' mode?

### DISCUSSION THREAD

**Discussion by ipstone** [helix-editor/helix#2844] [Q&A] [answered]
Hello, 
Coming from vim, I usually remap` <space>`+z to :quit - 
is there a way to do so in helix?

I tried to add the following in config.toml , but helix would not accept it:
```
[keys.space]
z = 'quit'
```

Appreciate help/ideas. Thanks

**Comment by the-mikedavis** [comment, marked as answer]
This should do the trick:

```toml
[keys.normal.space]
z = ":quit"
```

**Reply by D4ario0** [reply]
How would you add a description to the command?

For Instance I would like to indicate something like 'LazyGit Panel':
[keys.normal.space]
m = [
    ":write-all", 
    ":new", 
    ":insert-output lazygit", 
    ":set mouse false",
    ":set mouse true",
    ":buffer-close!",
    ":redraw", 
    ":reload-all"
]

I get:
![Screenshot_20250514_133341](https://github.com/user-attachments/assets/1abe6700-a99f-428a-a595-dbc353d3e825)
